CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Tabstops not being reset with `reset`
 - Selection not cleared when switching between main and alt grid
+- Resize lag on launch under some X11 wms
```

Comment:
Might be good to add another comment about input latency improvement by one frame on X11?

Reply:
Maybe something like this would work:

```
 - Increased input latency due to vsync behavior on X11
```